docs/concepts/config.md

```diff
@@ -67,7 +67,7 @@ data:
 
 - `ipamUnixSocketPath` (string): Spiderpool agent listens to this UNIX socket file and handles IPAM requests from IPAM plugin.
 - `networkMode`:
-  - `legacy`: Applicable to the traditional physical machine network.
+  - `legacy`: Applicable to the traditional network consisting of physical machines.
 - `enableIPv4` (bool):
   - `true`: Enable IPv4 IP allocation capability of Spiderpool.
   - `false`: Disable IPv4 IP allocation capability of Spiderpool.
```

Comment:
This is just a snippet of code configuration, so it's difficult to provide a complete review. However, based on the provided information, I can only suggest minor improvements:

- Use more descriptive variable names for better readability.
- Add comments to explain the purpose of each configuration option.
- Consider adding default values for configuration options that are not required, which can help avoid errors.
- Ensure that the code patch has been tested thoroughly before applying it in a production environment.

Overall, it seems like a straightforward configuration patch with no obvious bug risk.